src/spryker/harness.yml

```diff
@@ -95,6 +95,9 @@ attributes:
     host: jenkins
     port: 8080
     external_host: = 'jenkins-' ~ @('hostname')
+  jenkins_runner:
+    # used by the Jenkins JNLP agent
+    java_version: 8
```

Comment:
Can we switch this automatically like we do for the console/php-fpm image?

Reply:
if php version is 7.4 assume buster so assume 11? :)

Reply:
Ah we have 7.3 buster images too, hmm

Reply:
only latest spryker release supports php 7.4, which isn't supported by harness yet.

Reply:
yep, tho I've done this because I need to use the buster 7.3 image

Reply:
I'm not sure existing projects are held back in upgrading because of the harness tho (tho zed stuff still needs sorting out)

Reply:
Could do an attribute for base OS, but too complicated if you can just set this on the project. Approved :)

Reply:
Oh there is:
```
default-jre/oldstable 2:1.8-58+deb9u1 amd64
  Standard Java or Java compatible Runtime
```

This picks openjdk-8-jre on stretch and openjdk-11-jre on buster

Reply:
I would also wonder if default-jre-headless as well (no gui bits for the app)

Reply:
Potentially, though I guess PDF rendering tasks will need some X components/fonts. Unsure if Spryker does any of that :)